src/Text.Analyzers/Core/IdentifiersShouldBeSpelledCorrectly.cs

```diff
@@ -240,6 +240,7 @@ public override void Initialize(AnalysisContext context)
 
         private static void OnCompilationStart(CompilationStartAnalysisContext compilationStartContext)
         {
+
```

Comment:
Revert?

Reply:
done, I also removed the blank line at the end of the file.
apparently, the GitHub web editor adds a blank line at the end automatically, which is why there are three commits for the same thing...